code/game/objects/objs.dm

```diff
@@ -277,7 +277,11 @@
 
 //trying to buckle a mob
 /obj/proc/buckle_mob(mob/M, mob/user)
-	if (!ismob(M) || (get_dist(src, user) > 1) || user.is_mob_restrained() || user.stat || buckled_mob || M.buckled || !isturf(user.loc))
+	if (!ismob(M) || (get_dist(src, user) > 1) || user.is_mob_incapacitated() || user.stat || buckled_mob || M.buckled || !isturf(user.loc))
+		return
```

Comment:
Consider splitting this up to have `user.is_mob_incapacitated()` in the if block below so it has a message.